frontend/src/components/BookingCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import messageIcon from '../assets/messageIcon.svg'
+import adminIcon from '../assets/adminIcon.svg'
+
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import { RootState } from '../store/types'
+import { PATHS } from '../routes/Routes'
+import { DOMAIN_URL, ENDPOINTS } from '../store/endpoints'
+import { months, days } from '../common/dates'
+
+const BookingGroup = styled.div``
```

Comment:
if there is no styling for booking group consider using div instead of making a styled component. and if it does not require a div, consider using empty tags (`<></>`)

Reply:
Resolved in commit 6c4688c5a5c1853aa8ff09a0d49fdb749f87bdbb